example/build.gradle

```diff
@@ -18,8 +18,9 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            signingConfig signingConfigs.debug
```

Comment:
Added by Android Studio to allow the app to run in "release mode" on the emulator